pkg/util/validation/limits.go

```diff
@@ -128,6 +128,7 @@ type Limits struct {
 	// between the stored timestamp and the time we received a sample is
 	// more than this duration
 	HATrackerFailoverTimeout                    model.Duration      `yaml:"ha_tracker_failover_timeout" json:"ha_tracker_failover_timeout" category:"advanced" doc:"description=If we don't receive any samples from the accepted replica for a cluster in this amount of time we will failover to the next replica we receive a sample from. This value must be greater than the update timeout."`
+	HATrackerUseSampleTimeForFailover           bool                `yaml:"ha_tracker_use_sample_time_for_failover" json:"ha_tracker_use_sample_time_for_failover" category:"advanced" doc:"description=Use the sample time for failover instead of the current time. This is useful to prevent samples being too close together during failover when write requests are delayed such that the sample time is earlier than the current time."`
```

Comment:
do we need to explain this twice? isn't the flag below enough? it seems to be enough for `drop_labels`